CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+You can also post in our [technical](https://groups.google.com/forum/#!forum/archivematica-tech) or [user](https://groups.google.com/forum/#!forum/archivematica) mailing lists.
+A post to the mailing list is always welcome, especially if you're unsure if it's a bug or a local problem!
+
+Useful questions to answer if you're having problems include:
+
+* What version of metsrw are you using?
+* How was it installed?
+* What did you do to cause this bug to happen?
+* What did you expect to happen?
+* What did you see instead?
+* Can you reproduce this reliably?
+
+## Contribution standards
+
+For more information on contribution guidelines and standards, see the CONTRIBUTING.md in the [Archivematica project](https://github.com/artefactual/archivematica).
+
+## Contributor's Agreement
+
+In order for the Archivematica development team to accept any patches or code commits, contributors must first sign this [Contributor's Agreement](https://wiki.archivematica.org/images/2/25/Contributor_agreement.txt).
+The Archivematica contributor's agreement is based almost verbatim on the [Apache Foundation](http://apache.org )'s individual [contributor license](http://www.apache.org/licenses/icla.txt).
+
+If you have any questions or concerns about the Contributor's Agreement, please email us at agreement@artefactual.com to discuss them.
+
+### Why do I have to sign a Contributor's Agreement?
+
+One of the key challenges for open source software is to support a collaborative development environment while protecting the rights of contributors and users over the long-term.
+Unifying Archivematica copyrights through contributor agreements is the best way to protect the availability and sustainability of Archivematica over the long-term as free and open-source software.
+In all cases, contributors who sign the Contributor's Agreement retain full rights to use their original contributions for any other purpose outside of Archivematica, while enabling Artefactual Systems, any successor organization which may eventually take over responsibility for Archivematica, and the wider Archivematica community to benefit from their collaboration and contributions in this open source project.
+
+[Artefactual Systems](http://artefactual.com) has made the decision and has a proven track record of making our intellectual property available to the community at large.
+By standardizing contributions on these agreements the Archivematica intellectual property position does not become too complicated.
+This ensures our resources are devoted to making our project the best they can be, rather than fighting legal battles over contributions.
+
+### How do I send in an agreement?
+
+Please print out, read, sign, and scan the [contributor agreement](https://wiki.archivematica.org/images/2/25/Contributor_agreement.txt) and email it to agreement@artefactual.com
```

Comment:
Does it matter that this will be different from the link the other contributing.mds use?